entities/weapons/raft_builder/cl_init.lua

```diff
@@ -20,26 +35,33 @@ function SWEP:Initialize()
         local clsName = placeable.class
         local cls = baseclass.Get( clsName )
 
-        local mat = RVR.Util.getModelTexture( cls.Model, cls.PreviewPos, cls.PreviewAngle )
+        local mat = raftPartIcons[clsName]
+
         self.radial:AddItem( cls.PrintName, mat, function()
             self:SetSelectedClass( clsName )
         end )
     end
 
     local raftBuilder = self
     function self.radial:customPaint()
-        draw.NoTexture()
+        self:SetCenterOutlineColor()
         if not self.selectedItem then return end
+        draw.NoTexture()
         local className = raftBuilder.Placeables[self.selectedItem].class
         local class = baseclass.Get( className )
         local required = class:GetRequiredItems()
 
         for i, itemData in ipairs( required ) do
-            raftBuilder.drawItemRequirement( ScrW() / 2, ScrH() * 0.48 + i * 40, itemData.item.type, itemData.count, "DermaLarge" )
+            raftBuilder.drawItemRequirement( ScrW() * 0.497, ScrH() * 0.5 + ( i - 1 ) * 30,
+                itemData.item.type, itemData.count, "RVR_RaftBuilderIngredients", ScrH() * 0.025 )
```

Comment:
miiiight be nice to have at _least_ a comment here, but preferably calculating each param in its own named var so we can understand what's going on and how to modify it